test.properties

```diff
@@ -6018,6 +6019,61 @@
             (testray.main.component.name ~ "Password Policies")\
         )
 
+    #
+    # PEDS USM Transfer
+    #
+
+    test.batch.class.names.includes[peds-um]=\
+        **/feature-flag/**/*Test.java,\
+        **/portal-instances/**/*Test.java,\
+        **/portal-language-override-test/**/*Test.java
+
+    test.batch.dist.app.servers[peds-um]=tomcat
+
+    test.batch.names[peds-um]=\
+        functional-tomcat90-mysql57-jdk8,\
+        js-unit-jdk8,\
+        modules-integration-mysql57-jdk8,\
+        modules-unit-jdk8
+
+    test.batch.run.property.query[functional-tomcat90-mysql57-jdk8][peds-um]=\
+        (\
+            (portal.acceptance != quarantine) AND \
+            (portal.upstream == true)\
+        ) AND \
+        (\
+            (testray.main.component.name ~ "Instance Settings") OR \
+            (testray.main.component.name ~ "Language Override") OR \
+            (testray.main.component.name ~ "System Settings")\
+        )
```

Comment:
Is there maybe a better name we can give these group of components we inherited from USM team? I'm thinking PEDS System Management and renaming `peds-um` test suite to something like `ci:test:system-management` or `ci:test:peds-sm`. cc/ @ethib137 @pedro-michel @mateo-hermosin

Reply:
I updated this section as PEDS system management and abbreviated the test suite to `peds-sm`. Let me know if we decide on a better name. Will forward for now.